src/_playground/documentation/Import/Import.js

```diff
@@ -1,16 +1,17 @@
+import arraySort from 'array-sort';
+import packageJson from '../../../../package.json';
 import PropTypes from 'prop-types';
 import React from 'react';
 
-export const Import = (props) => {
-
-    const { module, path } = props;
+export const Import = ({ sourceModule }) => {
 
     return (
-        <code>import &#123; { module } &#125; from '{ path }';</code>
+        <code>
+            import &#123; { arraySort(Object.keys(sourceModule)).join(', ') } &#125; from '{packageJson.name}';
```

Comment:
In addition to fixing the path shown in the example `import` on each component's playground page, let's also remove hard-coded value props and just scan the code to find exported components and the name of the package.